fbw-common/src/systems/instruments/src/EFB/Dashboard/Widgets/ColorMetar.tsx

```diff
@@ -9,37 +9,34 @@ import { ColorCode, MetarParserType } from '@flybywiresim/fbw-sdk';
  * ColorCode: src/systems/instruments/src/Common/metarTypes.ts
  */
 export const ColoredMetar = ({ metar }: { metar: MetarParserType }) => {
-    const partsList = metar.raw_parts.map((metarPart, index) => {
-        let style = '';
+  const partsList = metar.raw_parts.map((metarPart, index) => {
+    let style = '';
 
-        switch (metar.color_codes[index]) {
-        case ColorCode.Highlight:
-            style = 'text-theme-highlight';
-            break;
-        case ColorCode.Info:
-            style = 'text-gray-500';
-            break;
-        case ColorCode.Caution:
-            style = 'text-yellow-500';
-            break;
-        case ColorCode.Warning:
-            style = 'text-red-400';
-            break;
-        case ColorCode.TrendMarker:
-            style = 'font-bold text-gray-500';
-            break;
-        default:
-        }
-
-        return (
-            <span key={metarPart} className={`text-2xl ${style}`}>
-                {metarPart}
-                {' '}
-            </span>
-        );
-    });
+    switch (metar.color_codes[index]) {
+      case ColorCode.Highlight:
+        style = 'text-theme-highlight';
+        break;
+      case ColorCode.Info:
+        style = 'text-gray-500';
+        break;
+      case ColorCode.Caution:
+        style = 'text-yellow-500';
+        break;
+      case ColorCode.Warning:
+        style = 'text-red-400';
+        break;
+      case ColorCode.TrendMarker:
+        style = 'font-bold text-gray-500';
+        break;
+      default:
+    }
 
     return (
-        <span>{ partsList }</span>
+      <span key={metarPart} className={`text-2xl ${style}`}>
+        {metarPart}{' '}
```

Comment:
I don't understand either the before or after, equally bad. Manually clean this up at some point...